server/cp/ai/semaphore.py

```diff
@@ -109,15 +118,14 @@ def fetch_parent_info(self, qcode):
             return []
 
     # Analyze2 changed name to analyze_parent_info
-    def analyze_parent_info(self, html_content: dict) -> dict:
+    def analyze_parent_info(self, html_content: str) -> dict:
```

Comment:
`html_content` can't be a `str` if it's used later as a `Dict`

Reply:
changed back to dict. resolved